src/julia_gcext.h

```diff
@@ -1,6 +1,10 @@
 #ifndef JL_GCEXT_H
```

Comment:
Also missing `// This file is a part of Julia. License is MIT: https://julialang.org/license` here, but we just should remember to do that for all files before the release (and to add it to our release-candidate checklist in the Makefile)